helm/eirini/values.yaml

```diff
@@ -28,3 +28,4 @@ env:
   # Base domain of the SCF cluster.
   # Example: "my-scf-cluster.com"
   DOMAIN: ~
+EIRINI_ROOTFS_VERSION: 
```

Comment:
Is there a reasonable default that can be placed here and updated by eirini-release maintainers?

Reply:
That has been kept intentionally blank. As we do not want to bits cluster to run if no value is passed by the user.